test/gtest/uct/ib/test_dc.cc

```diff
@@ -209,8 +214,10 @@ UCS_TEST_P(test_dc, dcs_multi) {
     }
 
     /* this should fail because there are no free dci */
-    status = uct_ep_am_short(m_e1->ep(i), 0, 0, NULL, 0);
-    EXPECT_EQ(UCS_ERR_NO_RESOURCE, status);
+    if (!uct_dc_mlx5_iface_is_hybrid(iface)) {
+        status = uct_ep_am_short(m_e1->ep(i), 0, 0, NULL, 0);
+        EXPECT_EQ(UCS_ERR_NO_RESOURCE, status);
```

Comment:
maybe we can move uct_ep_am_short() before (outside) the uct_dc_mlx5_iface_is_hybrid check?